### DISCUSSION TITLE
Hide Expand column untill there is at least 1 group

### DISCUSSION THREAD

**Discussion by kopach** [KevinVandy/material-react-table#42] [Q&A] [answered]
Currently, if I enable row grouping - "Expand" column will always be visible even though inactive initially. Is there any way to hide such "expand" column until user creates at least 1 group?

**Comment by KevinVandy** [comment]
This bug will be fixed in the next release. More grouping and aggregation features coming before 1.0 too

**Comment by KevinVandy** [comment, marked as answer]
This is fixed as of `v0.39.0`
